Week3/homework/step2-3.js

```diff
@@ -4,10 +4,13 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  if (num > 0) {
+    for (let i = 0; i < num; i++) {
+      result += str;
+    }
+  } else {
```

Comment:
You are not really returning anything useful so you could just skip the `else` here.